progress.md

```diff
@@ -0,0 +1,137 @@
+)# Progress
```

Comment:
This is a file I should've created from the start, thank you for doing this! Could you also create a `docs` directory and move it there? Also, seems that `)` in the beginning is not needed:

```suggestion
# Progress
```

Reply:
How detailed should this be? I originally had it in a list format which could’ve displayed individual method names but I figured the table would give people a good overview of what is usable and where they can jump in.

Reply:
Oh, what is currently there already looks great. Maybe I'd tweak table headers, they look kind of weird as empty thin rows, but don't sweat it. I think the API comparison should be automated at some point, but for the list of basic fundamental views this should be more than enough from my perspective, unless you think otherwise.

Hey @carson-katri, I'd be happy to hear your opinion on this, what do you think?

Reply:
I can explore an automated approach. In the meantime, you could do the table with HTML to get rid of the header if you wanted to fix them.

Reply:
I don't think headers are that important, so wouldn't mind if they are kept as is in this PR.